### DISCUSSION TITLE
Translating

### DISCUSSION THREAD

**Discussion by EntentaPL** [jeffalo/ocular#134] [Q&A] [answered]
Possible translating ocular? 
Thanks for answer :~)

**Comment by FunctionalMetatable** [comment]
It would be cool, but ocular isn't **that big** of Jeffalo's projects and it may be hard to implement translations?

**Comment by jeffalo** [comment, marked as answer]
sorry for missing this one, i completely forgot about discussions!

unfortunately i don't have the time/resources to maintain a set of translations and the code that supports it. :( ocular will stay in english only for the time being.
